### ISSUE TITLE
Allow updating and configuring other than YAML configuration formats (XML, PHP, Attributes)

### ISSUE THREAD

**Issue by kaznovac** [symfony/flex#970]
As a developer I would like to configure/update symfony app's configuration files with my choice of configuration file format.

`symfony/flex` should allow developer to state the format of the configuration files (or even try to detect them from the application at hand)

Lack of this feature greatly hinders usability of `flex`'s `recipes:update` for *non-yaml* configurations

related: #791

**Comment by arderyp** [comment]
Agreed, this would be slick, especially given compelling reasons for using php configs over yaml, which is suspect will become the new norm

https://getrector.com/blog/modernize-symfony-configs

**Comment by nicolas-grekas** [comment]
Closing as this is far from obvious in terms of impact + keeping an issue open won't help make it happen.

**Comment by arderyp** [comment]
If symfony docs and code are pushing us towards php configs, which is a good idea, then this should be addressed.
